common/js/web_vital_init.js

```diff
@@ -0,0 +1,29 @@
+function print(metric) {
+  const m = {
+    id: metric.id,
+    name: metric.name,
+    value: metric.value,
+    rating: metric.rating,
+    delta: metric.delta,
+    numEntries: metric.entries.length,
+    navigationType: metric.navigationType,
+    url: window.location.href,
+  }
+  window.sendWebVitalMetric(JSON.stringify(m))
+}
+
+async function load() {
+  let {
+    onCLS, onFID, onLCP, onFCP, onINP, onTTFB
+  } = await import('https://unpkg.com/web-vitals@3?module');
```

Comment:
Wouldn't this fetch the script from the remote? Can we incorporate it into our browser module? Or have we forgotten this file?

Reply:
+1

Reply:
It appears the file has not been forgotten. I've tested this with `fillform.js` and here's the output:

```console
# loading means load()
# printing means print()
INFO[0001] "loading..."                                  source=browser-console-api
INFO[0001] "printing..."                                 source=browser-console-api
INFO[0001] "printing..."                                 source=browser-console-api
INFO[0002] "printing..."                                 source=browser-console-api
INFO[0002] "printing..."                                 source=browser-console-api
INFO[0003] "printing..."                                 source=browser-console-api
INFO[0003] "printing..."                                 source=browser-console-api
INFO[0003] "loading..."                                  source=browser-console-api
INFO[0003] "printing..."                                 source=browser-console-api
INFO[0003] "printing..."                                 source=browser-console-api
INFO[0003] "printing..."                                 source=browser-console-api
INFO[0003] "printing..."                                 source=browser-console-api
INFO[0003] "printing..."                                 source=browser-console-api
INFO[0003] "printing..."                                 source=browser-console-api
INFO[0003] "loading..."                                  source=browser-console-api
```

Reply:
🤣  I can't believe i missed this detail. Resolved in 54c094d071f3e630864e0590de460f10057f90ce